managed/ui/src/components/config/Storage/StorageConfiguration.js

```diff
@@ -92,7 +92,12 @@ class StorageConfiguration extends Component {
     super(props);
 
     this.state = {
-      enableEdit: false,
+      enableEdit: {
+        s3: false,
+        nfs: false,
+        gcs: false,
+        az: false
```

Comment:
Isn't only one of these values ever true at a time? I think it makes more sense to have an `activeTab: string` as the state.

Reply:
Yes, only 1 value true at a time. This is just an initial state I defined but I'm using `activeTab` only when I'm setting up the state for the respective tab. You can check the `onEditConfig` method call.